nerfstudio/models/nerfacto.py

```diff
@@ -69,7 +69,7 @@ class NerfactoModelConfig(ModelConfig):
     """How far along the ray to stop sampling."""
     background_color: Literal["background", "last_sample"] = "last_sample"
     """Whether to randomize the background color."""
-    num_proposal_samples_per_ray: Tuple[int] = (256, 96)
+    num_proposal_samples_per_ray: Tuple[int, ...] = (256, 96)
```

Comment:
same here?

Reply:
This is where the typing issue comes from... in the run_actions script, you get typing errors when you run since the type of (256, 256) is Tuple[int, ...], not Tuple[int]